packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,248 @@
+# AutoComplete
+
+A SelectInput where you can type inside the input to filter through the items.
+
+<img width="215" alt="image" src="https://github.com/razorpay/blade/assets/30949385/63d95ab7-6231-4986-befb-1634269e7af3">
+
+## Design
+
+- [Figma - SelectInput & AutoComplete](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13590%3A171038&mode=dev)
+
+## Props
+
+> **Note**
+>
+> Below this section, I have added [examples of common AutoComplete usage](#usage). Check them out to understand usage of these props in details.
+
+| **Props**          | **Description**                                                                                                      | **Type**                                                        | **Default Value** |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- | ----------------- |
+| label              | label of input                                                                                                       | string                                                          |                   |
```

Comment:
the type here would be `React.ReactText | React.ReactText[]` right to allow use cases like 
`Status: ${selectedOptions.length} selected`

Reply:
Oh it won't take the entire string. It will just take the first part of it (the label part). The selection part will be appended on our end by us